app/views/admin/categories/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Categories - Admin</h1>
+
+<table class="text-center">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Parent category</th>
+      <th>Products Count</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @categories.each do |category| %>
+      <tr>
+        <td><%= link_to category.name, books_admin_category_path(category.id) %></td>
```

Comment:
pass category object instead of category.id
let me know the difference between two